sections/introduction.tex

```diff
@@ -0,0 +1,64 @@
+
+One particular field, which covers a variety of different use cases and applications, such as being useful for recommendations, is the field of topic modeling. \vejleder{gerne mere skarp i forhold til anbefalinger}
+Topic modeling has been used for finding topics within a collection of documents to either annotate or categorize certain collections of texts~\cite{Probabilistic_Topic_Models}.
+The \gls{lda} model is a well-cited topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}.
+Extensions of \gls{lda} have also been proposed to model various other information sources to generate better topics and/or topics, however, with different focuses and potential uses.
+In most models metadata is not used.\vejleder{er dette korrekt? + noget mere jeg ikke kan læse}
+Author-Topic model by \citet{author_topic_2012} and the MetaLDA model by \citet{MetaLDA2017} being notable examples.
+
+% Author-Topic
+The Author-Topic model by \citet{author_topic_2012} combines the \gls{lda} model with author information to model the relationship between authors and the documents, which they have written.
+This is based on the assumption that most authors usually write about only a few different topics.
+By modeling this relationship, connections between authors and topics, and between different authors, can be inferred giving more underlying information to the topics.
+They also show that the Author-Topic combination yields better and more coherent topics, which begs the question of whether any other document-related data can be applied similarly.
+However, they only test their algorithm on scientific papers, where usually the authors write about the same subject (their research field), which might not be the case for other fields, like journalism. \vejleder{Ikke helt holdbart! På den anden side så ser man nok mere spredning i ?? inden du nyheder + news + forfattere}
+
+% Intro to Nordjyske data and applications
+We have a dataset from a danish media group, called Nordjyske, with three years of article data.
+The dataset contains a variety of different metadata, which have the potential to improve topic models in the same way as \citet{author_topic_2012}. 
+In this paper, the dataset will be used with a focus on topic modeling and incorporation of metadata.
+
+% Intro to meta data
+The dataset used with a topic model can have a large impact on the topics generated.
+When referring to a dataset, we talk about a dataset including metadata information.
+One example of this could be the publication date of a text document. 
+In most contexts, it is not needed to use and understand the document; however, it provides more context to the main content of the document: the text.
+However, a lot of metadata is not used, either because it is not useful or we do not know how to use it properly. \vejleder{lidt ude af kontekst den her paragraf}
```

Comment:
```suggestion
However, a lot of metadata is not used, either because it is not useful or it is not known how to use it properly. \vejleder{lidt ude af kontekst den her paragraf}
```

Reply:
```suggestion
```

Reply:
```suggestion
```